src/crimson/net/ProtocolV2.cc

```diff
@@ -5,8 +5,11 @@
 
 #include <seastar/core/lowres_clock.hh>
 #include <fmt/format.h>
+#if __has_include(<fmt/chrono.h>)
```

Comment:
`__has_include` [is standardized since C++17](https://en.cppreference.com/w/cpp/preprocessor/include). Nice.